Doc/library/ast.rst

```diff
@@ -146,6 +146,77 @@ Node classes
     Snakes <https://greentreesnakes.readthedocs.io/en/latest/>`__ project and
     all its contributors.
 
+Root nodes
+^^^^^^^^^^
+
+.. class:: Module(body, type_ignores)
+
+   A Python module. Body contains a list of module's statements.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('x = 1'), indent=4))
+        Module(
+            body=[
+                Assign(
+                    targets=[
+                        Name(id='x', ctx=Store())],
+                    value=Constant(value=1))],
+            type_ignores=[])
+
+
+.. class:: Expression(body)
+
+   A single Python expression.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('123', mode='eval'), indent=4))
+        Expression(
+            body=Constant(value=123))
+
+
+.. class:: Interactive(body)
+
+   A single Python interactive statement, just like in :ref:`tut-interac`.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('x = 1; y = 2', mode='single'), indent=4))
+        Interactive(
+            body=[
+                Assign(
+                    targets=[
+                        Name(id='x', ctx=Store())],
+                    value=Constant(value=1)),
+                Assign(
+                    targets=[
+                        Name(id='y', ctx=Store())],
+                    value=Constant(value=2))])
+
+
+.. class:: FunctionType(argtypes, returns)
+
+   Special type to represent old-style type comments for functions.
+
+   Python versions prior to 3.6 used to annotate functions like this::
```

Comment:
I was a little confused here, even after several re-reads, because the sentences aren't explicitly connected and read somewhat disjointed to me. Furthermore, the "Python versions" themselves didn't annotate functions in a different style, rather older Python versions didn't support _users_ employing actual PEP 484 annotations, so instead users had to fall back to comments.

Also, weren't type annotations introduced in Python 3.5, per PEP 484 (PEP-484), not 3.6?

To try to fix these issues:

```suggestion
   Special type to represent old-style type comments for functions,
   as Python versions prior to 3.5 didn't support :pep:`484` annotations.

   Such type comments would look like this::
```